include/api/ofp_sysctl.h

```diff
@@ -54,6 +54,7 @@
 /*
  * Helper definitions
  */
+#undef __CONCAT
 #define	__CONCAT(x,y)	x ## y
```

Comment:
Isn't it possible to simply remove this define?

Reply:
That might be possible but then you must trust that __CONCAT is actually defined in some included header file and has an appropriate definition. I don't think this is a good idea.
Perhaps OFP should use its own definition of __CONCAT, e.g. OFP_CONCAT.

Reply:
> Perhaps OFP should use its own definition of __CONCAT, e.g. OFP_CONCAT.

Yes, I think this is the best option here. It should probably be _OFP_CONCAT, i.e. start with an underline to indicate an internal definition, not meant to be used by an application.

Reply:
Symbols starting with underscore are reserved. I think OFP_CONCAT is the better choice or possibly OFP__CONCAT.

Reply:
Regardless, leading underscore is used for this purpose in both OFP and ODP. (Even if this convention is not totally consistent in OFP.) OFP__CONCAT looks odd, since currently that prefix is used only with some OFP__*_T_DECLARED macros. OFP_CONCAT does not in any way convey that it's an internal definition.

Reply:
I can change it to e.g. _OFP_CONCAT.